cardano_node_tests/utils/dbsync_utils.py

```diff
@@ -1102,6 +1118,52 @@ def check_pool_data(ledger_pool_data: dict, pool_id: str) -> Optional[PoolDataRe
     return db_pool_data
 
 
+def check_pool_offline_data(
+    ledger_pool_data: dict, pool_id: str
+) -> Optional[PoolOfflineDataRecord]:
+    """Check comparison for pool offline data between ledger and db-sync."""
+    if not configuration.HAS_DBSYNC:
+        return None
+
+    # wait a bit for the dbsync thread that fills the PoolOfflineData table
+    time.sleep(60)
```

Comment:
I don't think this is good place for sleep, the function should check data in db, not trying to guess how much time passed since the data were updated in db. That is a concern of a test that calls this function.